pkg/scheduler/framework/plugins/interpodaffinity/filtering_test.go

```diff
@@ -807,7 +807,7 @@ func TestRequiredAffinitySingleNode(t *testing.T) {
 			node: &node1,
 			wantStatus: framework.NewStatus(
 				framework.UnschedulableAndUnresolvable,
-				"invalid label value",
+				`Invalid value: "{{.bad-value.}}"`,
```

Comment:
As a follow up, could you change the status comparison here to use cmp.Diff?

Reply:
Same as above, `PreFilter` also formats the error.